crate_universe/private/crate.bzl

```diff
@@ -34,7 +36,9 @@ def _spec(
         default_features (bool, optional): Maps to the `default-features` flag.
         features (list, optional): A list of features to use for the crate
         git (str, optional): The Git url to use for the crate. Cannot be used with `version`.
-        rev (str, optional): The git revision of the remote crate. Tied with the `git` param.
+        branch (str, optional): The git branch of the remote crate. Tied with the `git` param. Only one of branch, tag or rev may be specified.
```

Comment:
Can you update the docs for `branch` and `tag` to say something about `rev` being the recommended value?

Reply:
Done.